contracts/Safe.sol

```diff
@@ -444,8 +400,41 @@ contract Safe is
         address gasToken,
         address refundReceiver,
         uint256 _nonce
-    ) public view override returns (bytes32) {
-        return keccak256(encodeTransactionData(to, value, data, operation, safeTxGas, baseGas, gasPrice, gasToken, refundReceiver, _nonce));
+    ) public view override returns (bytes32 txHash) {
+        bytes32 separatorHash = domainSeparator();
```

Comment:
Why not:
```suggestion
        bytes32 domainSeparatorHash = domainSeparator();
```

Or


```suggestion
        bytes32 domainHash = domainSeparator();
```

In hashing, "domain" and "domain separator" is a common term used for additional data used to differentiate hashing output. I would slightly prefer to keep the word "domain" over "separator" (and would prefer most of all keeping both if it isn't too verbose).